content/en/logs/workspaces/sql_reference.md

```diff
@@ -0,0 +1,57 @@
+---
+title: SQL Reference
+further_reading:
+- link: "/logs/workspaces/"
+  tag: "Documentation"
+  text: "Learn more about Log Workspaces"
+---
+
+## Overview
+
+SQL in Analysis cells allows you to analyze and manipulate data within Log Workspaces. This documentation covers the SQL support available in Log Workspaces and includes:
+- [SQL functions](#functions)
+- [Syntax compatible with PostgreSQL](#syntax)
+
+
+{{< img src="/logs/workspace/sql_reference/sql_syntax_analysis_cell.png" alt="Example workspace cell with SQL syntax" style="width:100%;" >}}
+
+## Functions
+
+The following SQL functions are supported:
+
+| Function         | Description                                                        |
```

Comment:
For a function reference like this, I would suggest having more information about the actual signature of the functions (and/or specifying that these are just the same as the equivalent Postgres functions).

As an example of this (which could also be aligned/reused here) is the [existing DDSQL Editor docs](https://docs.datadoghq.com/ddsql_editor/reference/functions/), which provide the full signature, as well as having the split between scalar and aggregation functions (which is also very relevant).

Reply:
Thanks @gabrielreid, would you be able to provide an example, such as for the `MIN` function, what columns are you looking for and values.

Reply:
@gabrielreid I added examples for Syntax and Functions, is this close to what you're thinking?